lib/std/crypto/benchmark.zig

```diff
@@ -297,40 +297,40 @@ pub fn benchmarkAes8(comptime Aes: anytype, comptime count: comptime_int) !u64 {
 }
 
 const CryptoPwhash = struct {
-    hashFn: @compileError("anytype fields are removed from the language"),
-    params: @compileError("anytype fields are removed from the language"),
+    ty: type,
```

Comment:
The convention (https://ziglang.org/documentation/master/#Names) is to use `TitleCase` for anything that is a `type` or returns `type`.
In this case, usually `T` is used.
```suggestion
    T: type,
```

Reply:
The convention for fields, as far as I understand it, isn't affected by this which [you can see within existing structures](https://github.com/ziglang/zig/blob/master/lib/std/builtin.zig#L277) using fields of type `type`. Thus the lower case name is fine but would probably be better as `type` as it's a valid field name.

Reply:
That is interesting. I wonder if this is a mistake that got carried over repeatedly and thus went unnoticed or whether the docs need an update.